pkg/importer/util_test.go

```diff
@@ -136,6 +136,37 @@ var _ = Describe("Clean dir", func() {
 	})
 })
 
+var _ = Describe("Env vars to labels", func() {
+	It("Should convert KUBEVIRT_IO_ env vars to labels with values", func() {
+		envs := []string{
+			"INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE=u1.small",
+			"INSTANCETYPE_KUBEVIRT_IO_DEFAULT_PREFERENCE=fedora",
+		}
+		labels := envsToLabels(envs)
+		Expect(labels).To(HaveLen(2))
+		Expect(labels).To(HaveKeyWithValue("instancetype.kubevirt.io/default-instancetype", "u1.small"))
+		Expect(labels).To(HaveKeyWithValue("instancetype.kubevirt.io/default-preference", "fedora"))
```

Comment:
Can you please create an issue to update documentation, that tells us the difference between the usage of environment variables and labels? like we discussed offline in the meeting earlier today.

Reply:
Will do once this is merged.